app/_components/pdf-custom-renderer.tsx

```diff
@@ -81,32 +80,9 @@ export default function CustomRenderer() {
     }
     async function optionalContentConfigPromise(pdf: PDFDocumentProxy) {
```

Comment:
This is now only used for configuring how the pdf should be rendered, i.e. telling pdfjs which layers to show. Initializing which layers the document has now happens in PdfViewer.